cargo/private/cargo_utils.bzl

```diff
@@ -1,6 +1,7 @@
 """Utility functions for the cargo rules"""
 
 load("//rust/platform:triple_mappings.bzl", "system_to_binary_ext")
+load("//rust/private:utils.bzl", "abs")
```

Comment:
Seems I cannot depend on the private utils.bzl file from here. Should I just add the code here?

Reply:
I have just added a copy of the `abs` function to `cargo/private/cargo_utils.bzl`.